src/jit/codegenxarch.cpp

```diff
@@ -1367,6 +1367,15 @@ void CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
             {
                 assert(!isRegCandidate);
 
+#if defined(FEATURE_SIMD) && defined(_TARGET_X86_)
+                // Loading of TYP_SIMD12 (i.e. Vector3) variable
+                if (treeNode->TypeGet() == TYP_SIMD12)
+                {
+                    genLoadLclTypeSIMD12(treeNode);
+                    break;
+                }
+#endif // defined(FEATURE_SIMD) && defined(_TARGET_X86_)
```

Comment:
I would suggest to run LSRA stress mode runs as well after making changes to spill/reload code paths to handle TYP_SIMD12 locals.

---
In reply to: [87874711](https://github.com/dotnet/coreclr/pull/8109#discussion_r87874711) [](ancestors = 87874711)